src/components/Home/HomeHotPrices/HomeHotPrices.tsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from "react";
+import { useAppContext } from "../../../context/AppContext";
+import HomeHotPricesButtons from "./HomeHotPricesButtons";
+import Slider from "../../../UI/Slider";
+import useWindowSize from "../../../hooks/useWindowSize.hook";
+
+const HomeHotPrices = () => {
+  const { productsList } = useAppContext();
+  const [curElem, setCurElem] = useState<number>(0);
+  const itemsList = productsList.filter((el) => el.price / el.fullPrice < 0.75);
```

Comment:
change filtering for hot prices section so there would be more products than 3 watches ;)